homeassistant/components/fritzbox/binary_sensor.py

```diff
@@ -32,6 +29,22 @@ def __init__(self, device, fritz):
         self._device = device
         self._fritz = fritz
 
+    @property
+    def device_info(self):
+        """Return device specific attributes."""
+        return {
+            "name": self.name,
+            "identifiers": {(FRITZBOX_DOMAIN, self._device.ain)},
+            "manufacturer": self._device.manufacturer,
+            "model": self._device.productname,
+            "sw_version": self._device.fw_version,
+        }
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the device."""
+        return f"{self._device.ain}-{DOMAIN}"
```

Comment:
We don't need to add domain. The entity registry is aware of both the integration domain and the platform domain.

Reply:
Ok, the docs are not 100% accurate here:

https://developers.home-assistant.io/docs/en/next/entity_registry_index.html#unique-id-requirements

> If a device has a single serial but provides multiple entities, combine the serial with unique identifiers for the entities. For example, if a device measures both temperature and humidity, you can uniquely identify the entities using {serial}-{sensor_type}.

Reply:
That sounds accurate to me. The docs are talking about the case where the same platform has multiple different entities for the same device.

Reply:
I guess the docs should clarify that its only relevant within the same platform.